powertools-logging/src/main/java/software/amazon/lambda/powertools/logging/internal/LambdaLoggingAspect.java

```diff
@@ -305,49 +303,45 @@ private double samplingRate(final Logging logging) {
         return logging.samplingRate();
     }
 
+    @SuppressWarnings("java:S3457")
     private void logRequestHandlerEvent(final ProceedingJoinPoint pjp, final Object event) {
         Logger log = logger(pjp);
         if (log.isInfoEnabled()) {
-            LoggingUtils.logMessagesAsJson(true);
-            asJson(event).ifPresent(log::info);
-            LoggingUtils.logMessagesAsJson(false);
+            log.info("Handler Event", entry("event", event));
```

Comment:
for information, it will log:
```json
{
  "message": "Handler Event",
  "event": {
    // ...
  }
}
```
should we name it differently? should we log another message instead?